profiles/definitions/latest-2019-04-30.md

```diff
@@ -41,6 +41,21 @@ profiles:
           - managedClusters/accessProfiles
         '2017-07-01':
           - containerServices
+      microsoft.blockchain:
```

Comment:
could you help to explain why you need to update this file?

Reply:
what is this blockchain used for?